nginx.conf

```
@@ -52,6 +52,7 @@ http {
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/about/network/$                               $scheme://$http_host/docs/$1/network/overview/                              permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/curating/$                                    $scheme://$http_host/docs/$1/network/curating/                              permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/delegating/$                                  $scheme://$http_host/docs/$1/network/delegating/                            permanent;
+        rewrite ^/docs/([a-zA-Z][a-zA-Z])/network-transition-faq/$                      $scheme://$http_host/docs/$1/network/arbitrum-faq/                          permanent;
```

Looks like https://thegraph.com/docs/en/network/arbitrum-faq/ is a 404, did you mean
```suggestion
        rewrite ^/docs/([a-zA-Z][a-zA-Z])/network-transition-faq/$                      $scheme://$http_host/docs/$1/arbitrum/arbitrum-faq/                          permanent;
```
(https://thegraph.com/docs/en/arbitrum/arbitrum-faq/)?

---

Yes, omg, thank you. Could not figure that out for the life of me lol.